composer.json

```diff
@@ -50,7 +50,10 @@
             "bamarni/composer-bin-plugin": true,
             "composer/package-versions-deprecated": true
         },
-        "sort-packages": true
+        "sort-packages": true,
+        "platform": {
+            "php": "7.4.32"
```

Comment:
This should not have been added here. It is needed only in the vendor-bin composer.json files, to ensure correct lock file generation.

Reply:
@GrahamCampbell 

Feel free to open a pull request!